client/Android/Studio/freeRDPCore/src/main/res/values-zh_rTW/strings.xml

```diff
@@ -0,0 +1,252 @@
+    <string name="menu_sys_keyboard">鍵盤</string>
+    <string name="menu_ext_keyboard">功能鍵</string>
+    <string name="menu_touch_pointer">觸控指標</string>
+    <string name="menu_home">首頁</string>
+    <string name="menu_disconnect">中斷連線</string>
+    <!-- List section headers -->
+    <string name="section_bookmarks">手動連線</string>
+    <string name="section_active_sessions">使用中的連線</string>
+    <!-- Search strings -->
+    <string name="search_hint">連線到電腦</string>
+    <!-- List placeholder labels -->
+    <string name="list_placeholder_login">登入</string>
+    <string name="list_placeholder_no_servers">無遠端電腦</string>
+    <string name="list_placeholder_connecting">連線中 &#8230;</string>
+    <string name="list_placeholder_disconnecting">正在中斷連線 &#8230;</string>
+    <string name="list_placeholder_connection_error">連線失敗</string>
+    <string name="list_placeholder_wrong_password">密碼錯誤</string>
+    <string name="list_placeholder_invalid_username">無效的使用者名稱</string>
+    <string name="list_placeholder_add_bookmark">新增連線</string>
+    <!-- Bookmark settings strings -->
+    <string name="settings_cat_host">主機</string>
+    <string name="settings_label">標籤</string>
+    <string name="settings_hostname">主機名稱</string>
+    <string name="settings_port">連接埠</string>
+    <string name="settings_cat_credentials">登入資訊</string>
+    <string name="settings_credentials">登入資訊</string>
+    <string name="settings_username">使用者名稱</string>
+    <string name="settings_password">密碼</string>
+    <string name="settings_domain">網域</string>
+    <string name="settings_cat_settings">設定</string>
+    <string name="settings_screen">螢幕</string>
+    <string name="settings_cat_screen">螢幕設定</string>
+    <string name="settings_colors">顏色</string>
+    <string-array name="colors_array">
+        <item>256 色（8 位元）</item>
+        <item>高彩（15 位元）</item>
+        <item>高彩（16 位元）</item>
+        <item>全彩（24 位元）</item>
+        <item>最高品質（32 位元）</item>
+    </string-array>
+    <!-- This array is used as is in code,
+	do not translate! -->
+    <string-array name="colors_values_array" translatable="false">
+        <item>8</item>
+        <item>15</item>
+        <item>16</item>
+        <item>24</item>
+        <item>32</item>
+    </string-array>
+    <string name="settings_resolution">解析度</string>
+    <string name="resolution_automatic">自動</string>
+    <string name="resolution_fit">符合螢幕</string>
+    <string name="resolution_custom">自訂</string>
+    <string-array name="resolutions_array">
+        <item>自動</item>
+        <item>符合螢幕</item>
+        <item>自訂</item>
+        <item>640x480</item>
+        <item>720x480</item>
+        <item>800x600</item>
+        <item>1024x768</item>
+        <item>1280x1024</item>
+        <item>1440x900</item>
+        <item>1920x1080</item>
+        <item>1920x1200</item>
+    </string-array>
+    <!-- This array is used as is in code,
+	do not translate! -->
+    <string-array name="resolutions_values_array" translatable="false">
```

Comment:
please remove all elements that have `translatable="false"` set.
they are not used in UI elements but as identifiers in code.

Reply:
No problem, just I saw that they are inside some locales, so didn't notice that they remove be removed :sweat_smile: